CODEOWNERS

```diff
@@ -97,6 +97,7 @@
 /boards/arm/mimxrt*/doc/                  @MaureenHelm @MeganHansen
 /boards/arm/mps2_an385/                   @fvincenzo
 /boards/arm/msp_exp432p401r_launchxl/     @Mani-Sadhasivam
+/boards/arm/npcx7m6fb_evb/                @MulinChao
```

Comment:
> Fixed typo in Kconfig.npcx files. This CL also arranges the owners for
the npcx ec driver files made by nuvoton team in CODEOWNERS file.

These changes are orthogonal to one another, please split them into separate commits

Reply:
Done.